waterlog/src/Components/DailyWastage.js

```diff
@@ -29,15 +34,21 @@ const DailyWastageComponent = (props) => {
         ticks: {
           major: {
             fontStyle: 'bold',
-            fontColor: '#FF0000'
+            fontColor: '#919191',
```

Comment:
Question, why are you using hex here?
line 17-19 is rgb/rgba
i suggest stick to one standard. either rgba or hex. not all 3

Reply:
https://cssgenerator.org/rgba-and-hex-color-generator.html

Reply:
based on your project and your other files, stick to one convention accross all files. its bad to have hex, rgba, rgb

Reply:
also remove floating comma at the end